### ISSUE TITLE
Other Guests using hostsupdater

### ISSUE THREAD

**Issue by cd2-public** [cogitatio/vagrant-hostsupdater#77]
this would be super nice-to-have. Right now I run my main machine in Windows, but have a permanent VM which I run through. I Run my vagrant via my host as a guest in a guest seems insane!

Because of this, I'd love a way to output hosts entries in a folder so I can sudo cat filename >> /etc/hosts and use hosts redirects in my other guests.

I realize this is a big-ask, it's complicated, but it would be awesome to have

**Comment by cgsmith** [comment]
@cd2-public are you saying that you'd like to have vagrant-hostsupdater run on a guest VM?

**Comment by Lewiscowles1986** [comment]
no; I would like the ability to define for groups of VM's hostnames, which hostupdater will update on the host. When setting the hosts on each machine, I would like it to inject all hosts, with their static IP to the guests, so that guests can access each other via hostname as well as IP address
